src/wizard/android/index.md

```diff
@@ -5,10 +5,20 @@ support_level: production
 type: framework
 ---
 
+> Want to play with some new features? Try out our beta [Auto-Installation](#auto-installation-with-the-sentry-android-gradle-plugin) feature.
```

Comment:
It does not work, you have to add the full url

Reply:
No bloody idea how I can add a full url to this page, probably I can't?

Reply:
I can't find an example of linking to the heading within the wizard page in the any of the wizard files. I think just do this:

```suggestion
> Want to play with some new features? Try out our beta Auto-Installation feature documented below.
```

Reply:
Sounds good, I think I would reorder the entries then (so, auto-installation over the manual one). Since the auto-installation one has a beta note in it, it's prominent enough that it's not stable I think